Pod/Classes/WPMediaPickerViewController.m

```diff
@@ -624,37 +653,68 @@ - (UILabel *)defaultEmptyView
     return _defaultEmptyView;
 }
 
+- (void)addEmptyViewToView
+{
+    if (self.emptyView != nil && self.emptyView.superview != nil) {
+        return;
+    }
+
+    [self.emptyView setTranslatesAutoresizingMaskIntoConstraints:NO];
+    [self.containerEmptyView addSubview:self.emptyView];
+    
+    [NSLayoutConstraint activateConstraints:
+     @[
+       [self.emptyView.centerYAnchor constraintEqualToAnchor:self.containerEmptyView.centerYAnchor],
+       [self.emptyView.centerXAnchor constraintEqualToAnchor:self.containerEmptyView.centerXAnchor]
+       ]
+     ];
+}
+
 #pragma mark - Empty View Controller support
 
 - (void)addEmptyViewControllerToView
 {
-    if (!self.emptyViewController && self.emptyViewController.view.superview != nil) {
+    if (self.emptyViewController != nil && self.emptyViewController.view.superview != nil) {
         return;
     }
 
     [self addChildViewController:self.emptyViewController];
     [self.emptyViewController.view setTranslatesAutoresizingMaskIntoConstraints:NO];
-    [self.collectionView addSubview:self.emptyViewController.view];
-
-    self.emptyViewBottomConstraint = [self.emptyViewController.view.bottomAnchor constraintEqualToAnchor:self.view.bottomAnchor];
+    [self.containerEmptyView addSubview:self.emptyViewController.view];
 
     [NSLayoutConstraint activateConstraints:
      @[
-       [self.emptyViewController.view.topAnchor constraintEqualToAnchor:self.collectionView.topAnchor],
-       self.emptyViewBottomConstraint,
-       [self.emptyViewController.view.leadingAnchor constraintEqualToAnchor:self.view.leadingAnchor],
-       [self.emptyViewController.view.trailingAnchor constraintEqualToAnchor:self.view.trailingAnchor]
+       [self.emptyViewController.view.topAnchor constraintEqualToAnchor:self.containerEmptyView.topAnchor],
+       [self.emptyViewController.view.bottomAnchor constraintEqualToAnchor:self.containerEmptyView.bottomAnchor],
+       [self.emptyViewController.view.leadingAnchor constraintEqualToAnchor:self.containerEmptyView.leadingAnchor],
+       [self.emptyViewController.view.trailingAnchor constraintEqualToAnchor:self.containerEmptyView.trailingAnchor]
        ]
      ];
 
     [self.emptyViewController didMoveToParentViewController:self];
 }
 
+- (void)removeContainerEmptyView
```

Comment:
Can he hide/show the container instead of removing and adding it? It seems more convenient and will simplify the code. But I'm not sure it has drawbacks. What do you think?

Reply:
Let me do some tests with this behavior for hide/show, I don't think we have drawbacks to follow with it.

Reply:
Worked smoothly 😄, resolved here https://github.com/wordpress-mobile/MediaPicker-iOS/pull/399/commits/eaf7f546a638a29a556d00d97ec15941d1e4492d